rocky/reports/views/base.py

```diff
@@ -71,7 +74,13 @@ def setup(self, request, *args, **kwargs):
         self.selected_report_types = request.GET.getlist("report_type", [])
 
     def get_oois(self) -> List[OOI]:
```

Comment:
This will conflict later, depends on merging prio with:

https://github.com/minvws/nl-kat-coordination/blob/23f95f95575d4c92389f58aaf558843414073b01/rocky/reports/views/base.py#L75-L84